samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/QuestionScoreListener.java

```diff
@@ -228,12 +228,10 @@ public boolean questionScores(String publishedId, QuestionScoresBean bean,
 			}
 			// build a hashMap (publishedItemId, publishedItem)
 			Map<Long, ItemDataIfc> publishedItemHash = pubService.preparePublishedItemHash(publishedAssessment);
-			log.debug("questionScores(): publishedItemHash.size = "
-					+ publishedItemHash.size());
+			log.debug("questionScores(): publishedItemHash.size = {}", publishedItemHash.size());
 			// build a hashMap (publishedItemTextId, publishedItemText)
 			Map<Long, ItemTextIfc> publishedItemTextHash = pubService.preparePublishedItemTextHash(publishedAssessment);
-			log.debug("questionScores(): publishedItemTextHash.size = "
-					+ publishedItemTextHash.size());
+			log.debug("questionScores(): publishedItemTextHash.size = {}", publishedItemTextHash.size());
```

Comment:
```suggestion
			log.debug("publishedItemTextHash.size = {}", publishedItemTextHash.size());
```
the method name is redundant in the log message